README.md

```diff
@@ -44,32 +43,38 @@ optional arguments:
   --version             Display version information and dependencies.
   --verbose, -v, -d, --debug
                         Display extra debugging information and metrics.
-  --rank, -r            Present websites ordered by their Alexa.com global
-                        rank in popularity.
-  --folderoutput FOLDEROUTPUT, -fo FOLDEROUTPUT
-                        If using multiple usernames, the output of the results
-                        will be saved at this folder.
-  --output OUTPUT, -o OUTPUT
-                        If using single username, the output of the result
-                        will be saved at this file.
   --tor, -t             Make requests over TOR; increases runtime; requires
```

Comment:
I don't get why --rank, --folderoutput  and --json were removed

Reply:
Slight oversight on my part... added them back :)